### PR TITLE
Feature/330 animated hero icons

### PR DESCRIPTION
**Pull request by HaseebUllahAbbasi**

### CODE REVIEW

File: pnpm-lock.yaml
Comment (theblockstalk) [inline pnpm-lock.yaml:1]: this file should not be included I think

File: pages/_document.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ class MyDocument extends Document {
     return (
       <Html lang="zxx">
         <Head>
-          <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+          <meta httpEquiv="Content-Type" content="text/html; charset=utf-8" />
```
Comment (theblockstalk) [inline pages/_document.js:8]: should this change like this?

File: components/Home/MainBanner.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    height: "100%",
+    background: "rgba(0, 0, 0, 0.2)", // Adjust opacity here
+    zIndex: 1,
+  },
+  background: {
+    width: "100%",
+    height: "100%",
+    opacity: "0.5",
+    objectFit: "cover",
+    zIndex: -10,
```
Comment (theblockstalk) [inline components/Home/MainBanner.js:29]: this is the desired design for how it blends to the background.
https://drive.google.com/file/d/1JOMmk01skQTtlC7cRI2nvDajPuKJLvlS/view

I don't know how to do this but some kind of "radial gradient" sounds right. See https://stackoverflow.com/questions/32059991/how-to-make-a-radial-gradient-with-transparency-in-css

File: components/Home/MainBanner.js
Comment (theblockstalk) [inline components/Home/MainBanner.js:1]: animation does not match design https://drive.google.com/file/d/1JOMmk01skQTtlC7cRI2nvDajPuKJLvlS/view

- lightblue background (img_5) currently rotates 180 degrees and should only be 90 degrees
- padlock (img_0) and padlock top (img_3) should both slide in from top at same time. they also shouldn't zoom into frame at all.
- padlock top (img_3) should vertical flip from left to right after it slides in from top
- profile icon (img_2), then "ID" svg, then tonomy logo (img_1) should slide up into the middle of the circle, and then out, in order and replacing the previous image

this logic is all inside the demo/data.html file so you could copy most of it from there. What you have done is much cleaner and more controllable so if you can get it does with your own CSS and .js that is better. But i suggest the easiest strategy.

**Review by theblockstalk — CHANGES_REQUESTED** [review]

File: pages/_document.js (hunk 1 above)
Comment (HaseebUllahAbbasi) [reply pages/_document.js:8]: Yeah, i was being warned

File: pnpm-lock.yaml
Comment (HaseebUllahAbbasi) [reply pnpm-lock.yaml:1]: I used this svg as an image here

Comment (HaseebUllahAbbasi) [reply pnpm-lock.yaml:1]: that is removed ..

File: .gitignore [hunk 3]
```diff
@@ -66,4 +66,5 @@ out
 # dotenv environment variables file
 .env
 
-build
\ No newline at end of file
+build
+# pnpm-lock.yaml
```
Comment (theblockstalk) [inline .gitignore:70]: remove comment

File: components/Home/MainBanner.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import FsLightbox from "fslightbox-react";
+import Lottie from "lottie-react";
+import animatedData from "../assets/animated.json"; // Adjust the path as needed
```
Comment (theblockstalk) [inline components/Home/MainBanner.js:5]: lets call it assets/home/main-banner.js

File: components/Home/MainBanner.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    justifyContent: "center",
+    alignItems: "center",
+    width: "250px",
+    height: "250px",
+    borderRadius: "100px",
+    overflow: "hidden",
+    margin: "auto", // Center the entire div horizontally
+  },
+};
+const AnimatedHeroImage = () => {
```
Comment (theblockstalk) [inline components/Home/MainBanner.js:28]: add new lines before and after root level constants, styles, functions or functional components

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**